### ISSUE TITLE
Is https://github.com/joomla/joomla-cms/pull/39721 not merged correctly?

### ISSUE THREAD

**Issue by astridx** [joomla/joomla-cms#39853]
### Steps to reproduce the issue

Clone the branch 4.3-dev of this repo and open the file `.gitignore`.

### Expected result

`.gitignore` should look like in https://github.com/joomla/joomla-cms/pull/39721

### Actual result

The text `output `is not there. See https://github.com/joomla/joomla-cms/blob/c28c8e3db350a8598652771a388b37b626547ed4/.gitignore#L106

Although https://github.com/joomla/joomla-cms/pull/39721 has been merged, it does not appear in the historie of the file `.gitignore`.

**Comment by brianteeman** [comment]
Your pull request #39722 was merged after #39721 and reverted the change

**Comment by wojtekxtx** [comment]
@brianteeman why was this change reverted?

**Comment by HLeithner** [comment]
@astridx as you reverted it your self you have to decide which is the right one ;-) looking into the config file we use the output path

```json
  videosFolder: 'tests/cypress/output/videos',
  screenshotsFolder: 'tests/cypress/output/screenshots',
```

Can you please fix it again or did @laoneo see a valid reason why the .gitígnore has been changed in #39722 ?

**Comment by fancyFranci** [comment]
I assume wrong merge conflict resolving @wojtekxtx.
@astridx can you create a new PR to bring back the correct configs please?

**Comment by wojtekxtx** [comment]
@fancyFranci Thanks for reply 👍 

Instead of creating new PR, would not be easier to just `git reset --hard` to specific commit? Just thinking........

**Comment by laoneo** [comment]
That won't work as all the clones are messed up then.

**Comment by astridx** [comment]
See https://github.com/joomla/joomla-cms/pull/39890
